src/Reenbit.ChuckNorris.Domain/Entities/JokeCategory.cs

```diff
@@ -0,0 +1,17 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Text;
+
+namespace Reenbit.ChuckNorris.Domain.Entities
+{
+    public class JokeCategory
+    {
+        public int Id { get; set; }
+
+        public int JokeId { get; set; }
+        public Joke Joke { get; set; }
```

Comment:
each property should be separated by new line

Reply:
+